hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/common/BufferPool.java

```diff
@@ -236,11 +244,15 @@ public synchronized void close() {
       }
     }
 
+    int currentPoolSize = this.pool.numCreated();
```

Comment:
how about you cut the `this.` here and from the existing calls in the same method. not our project's style